openjdk-integ-tests/src/test/java/org/conscrypt/javax/net/ssl/SSLSessionTest.java

```diff
@@ -418,4 +422,54 @@ public void test_SSLSession_valueIndependence() {
             s.close();
         }
     }
+
+    private static String alterOriginalHostName(InetAddress inetAddress, String originalHostName)
```

Comment:
Just curious ... I see the removal of `AbstractSSLTest`.  Was this done to get us closer to the Android code?

Reply:
It's to make it so that the setting up of the provider happens outside the test.  That lets us reuse the test class with no modifications in Android and other places (where the provider doesn't need to be explicitly installed because it's part of the OS or we're testing Google Play Services' copy or whatever) by just swapping out the class that sets up the provider.

Reply:
Got it ... makes sense.